task_DEV_2/task_DEV_2/Program.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+
+namespace task_DEV_2
+{
+
+
+    class Program
+    {
+        static void Main(string[] args)
+        {
+
+            Console.WriteLine("Enter the size of the array :");
+
+            int arrayLength = Convert.ToInt32(Console.ReadLine());
+
+
+            int[] numberArray = new int[arrayLength];
+            for (int i = 0; i < arrayLength; i++)
+            {
+                if (arrayLength >= 20)
+
+                {
+                    break;
+
+                }
+                Console.WriteLine("array cannot be <= 20");
```

Comment:
Do you see this message on the screen when you enter length more then 20?

Reply:
No.